### PR TITLE
completed PostComments

### PR DESCRIPTION
**Pull request by Cnmoosavinia**

### CODE REVIEW

File: src/components/comments.jsx [hunk 1]
```diff
@@ -1,20 +1,57 @@
...
 
-function Comments({ comments, setComments, loading, setLoading }) {
+function Comments({ comments, setComments }) {
   const { review_id } = useParams();
+  const [newComment, setNewComment] = useState("");
+  const [loadingComments, setLoadingComments] = useState(true);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    postComment(newComment, review_id).then(() => {
```
Comment (callumu21) [inline src/components/comments.jsx]: What happens if post comment fails? How can we let a user know that something went wrong? Also, this current allows a user on a slow connection to make multiple post requests before the initial one has succeeded. You should prevent this with a condition in state.

File: src/api.js [hunk 2]
```diff
@@ -35,3 +35,19 @@ export const unlikeReview = (review_id) => {
     return res.data.review;
   });
 };
+
+export const postComment = (comment, review_id) => {
+  const inputComment = { username: "tickle122", body: comment };
```
Comment (callumu21) [inline src/api.js]: The username should be pulled from the currently logged in user. I know Cnmoosavinia doesn't exist in the database at the moment, but you may want to add him in the backend to allow this functionality to be demonstrated on the front-end.

File: src/components/comments.jsx [hunk 3]
```diff
@@ -1,20 +1,57 @@
...
+
   return (
     <ul className="comment-list">
+      <div className="comment-section">
+        <img
+          className="pfp"
+          src="https://scontent.fman4-2.fna.fbcdn.net/v/t1.18169-9/1044725_10151527841570922_2110461867_n.jpg?_nc_cat=108&ccb=1-7&_nc_sid=09cbfe&_nc_ohc=qzcXY7CFmlcAX9SVbW9&_nc_ht=scontent.fman4-2.fna&oh=00_AfDNLFtojnACZzG2HJZ47hgd88pBRCoEIMZysUuWNBs2Cw&oe=63B7EA0B"
+        ></img>
+        <form onSubmit={handleSubmit}>
+          <input
```
Comment (callumu21) [inline src/components/comments.jsx]: This is a single line input box. I'd look at using textarea to allow multi-line comments 👍

**Review by callumu21 — CHANGES_REQUESTED** [review]
Good start, Cyrus. Just missing the following features:

1. Prevent users from making multiple post requests before the first has been completed
2. Let the user know that something went wrong on an unsuccessful post
3. Allow a user to make multi-line comments (see my comment below for advice)

File: src/components/comments.jsx (hunk 1 above)
Comment (Cnmoosavinia) [reply src/components/comments.jsx]: I have made it so the textbox is readonly during the load so they cannot select it and the button is disabled. When comment is posted it goes back to normal. I am not sure how to process error hnadling yet but I might wait till the error ticket if that is okay so I can go through the whole application and deal with errors as one ticket?

File: src/api.js (hunk 2 above)
Comment (Cnmoosavinia) [reply src/api.js]: accessed api to get list of users then set current user to position [1] in the user array

File: src/components/comments.jsx (hunk 3 above)
Comment (Cnmoosavinia) [reply src/components/comments.jsx]: added text area and removed the ability to resize it. Textbox will scroll down if users write too much. may chance that with css later

**Comment by pkaodev** [comment]
Looks really good! Merge  away

**Pull request merged**